R/utils_render_reigster_html.r

```diff
@@ -112,6 +109,22 @@ generate_html_postfix_hrefs_reg <- function(filter, register_table_name) {
   return(hrefs)
 }
 
+generate_html_postfix_hrefs_non_reg <- function(filter, register_table_name){  
+  if (!(register_table_name %in% list("all_venues", "codecheckers"))){
```

Comment:
I needed a moment to understand this - how about not negating the check, but switching the content of the two cases, for better readability?

Reply:
Thanks, that makes sense. Incorporated the changes!